navigation/app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/MockNavigationActivity.java

```diff
@@ -191,6 +191,30 @@ public void onFailure(Call<DirectionsResponse> call, Throwable throwable) {
    */
 
   @Override
+  public void onMilestoneEvent(RouteProgress routeProgress, String instruction, int identifier) {
+    Timber.d("Milestone Event Occurred with id: %d", identifier);
+    switch (identifier) {
+      case NavigationConstants.URGENT_MILESTONE:
+        Toast.makeText(this, "High Milestone", Toast.LENGTH_LONG).show();
+        break;
+      case NavigationConstants.IMMINENT_MILESTONE:
+        Toast.makeText(this, "Medium Milestone", Toast.LENGTH_LONG).show();
+        break;
+      case NavigationConstants.NEW_STEP_MILESTONE:
+        Toast.makeText(this, "New Step", Toast.LENGTH_LONG).show();
+        break;
+      case NavigationConstants.DEPARTURE_MILESTONE:
+        Toast.makeText(this, "Depart", Toast.LENGTH_LONG).show();
+        break;
+      case NavigationConstants.ARRIVAL_MILESTONE:
+        Toast.makeText(this, "Arrival", Toast.LENGTH_LONG).show();
+        break;
+      default:
```

Comment:
I added the default for checkstyle to be happy. Looking back, it makes more sense to show a toast for the cases when the user adds a new milestone but doesn't do anything with it inside the switch statement.